### DISCUSSION TITLE
Keyboard shortcut "gd": go to dashboard or discussion?

### DISCUSSION THREAD

**Discussion by hollasch** [community/community#2906] [Discussions] [answered]
From the notifications page, the `gd` keyboard shortcut takes you to the dashboard (which I still think needs some work, by the way). From a project page, hitting `?` will display keyboard shortcuts, which also indicate that `gd` will take you to the dashboard page. However, with discussions enabled, `gd` _actually_ takes you to the discussions page. Needs to be sorted out.

**Comment by becca** [comment, marked as answer]
This is a bug we are tracking to resolve. We'll let you know when we land a fix!

**Reply by becca** [reply]
This issue should now be fixed. We'll be updating the documentation shortly but wanted to let you know that `gd` lands you on the dashboard and we'll be adding `gg` for Discussions

**Reply by hollasch** [reply]
Thanks!
